### PR TITLE
Removed deprecated getMock and setExpectedException methods

### PR DESCRIPTION
**Pull request by phpanarchist**

### CODE REVIEW

**Comment by carusogabriel** [comment]
As we need to keep compatibility with `PHPUnit 4`, `createMock` shouldn't be used. Use `getMockBuilder($class)->getMock()` instead.

File: tests/unit/CommonMarkConverterTest.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function testConfigOnlyConstructor()
     public function testEnvironmentAndConfigConstructor()
     {
         $config = ['foo' => 'bar'];
-        $mockEnvironment = $this->getMock('League\CommonMark\Environment');
+        $mockEnvironment = $this->createMock('League\CommonMark\Environment');
```
Comment (carusogabriel) [inline tests/unit/CommonMarkConverterTest.php]: As we need to keep compatibility with `PHPUnit 4`, `createMock` shouldn't be used. Use `getMockBuilder($class)->getMock()` instead.

File: tests/unit/EnvironmentTest.php [hunk 2]
```diff
@@ -91,7 +91,7 @@ public function testSetConfig()
 
     public function testSetConfigAfterInit()
     {
-        $this->setExpectedException('RuntimeException');
+        $this->expectException('RuntimeException');
```
Comment (carusogabriel) [inline tests/unit/EnvironmentTest.php]: As we need to keep `PHPUnit 4` compatibility, `expectException` shouldn't be used. Use `@expectedException` annotation instead.

**Comment by colinodell** [comment]
I'll be dropping support for PHPUnit 4 in a few minutes :)

Thanks for the contribution!

**Pull request merged**